src/doc/reference.md

```diff
@@ -3888,7 +3883,7 @@ Box<i32>, y: Box<i32>)` declare one mutable variable `x` and one immutable
 variable `y`).
 
 Methods that take either `self` or `Box<Self>` can optionally place them in a
-mutable slot by prefixing them with `mut` (similar to regular arguments):
+mutable variable by prefixing them with `mut` (similar to regular arguments):
```

Comment:
Actually I think the original "slot" might be more accurate.
If I have to change "slot" to something I would use "binding" in this case.
See https://github.com/rust-lang/rust/pull/24149